server/routes/userRoutes.js

```diff
@@ -1,9 +1,17 @@
 const express = require("express");
 const router = express.Router();
 
-const { registerUser, loginUser } = require("../controllers/userController");
+const {
+  registerUser,
+  loginUser,
+  logoutUser,
+  makeUserAChef,
+} = require("../controllers/userController");
+const { auth } = require("../middlewares/authMiddlewares");
 
 router.route("/register").post(registerUser);
 router.route("/login").post(loginUser);
+router.route("/logout").get(logoutUser);
+router.route("/isChef").put(auth, makeUserAChef);

```

Comment:
maybe rename this, I would assume this route would return whether or not a user is a chef, not make the user a chef